inst/sql/sql_server/result_dataframe_ddl.sql

```diff
@@ -1,9 +1,9 @@
 --DDL to create dqdashboard_results table.
 
-IF OBJECT_ID('@resultsDatabaseSchema.@tableName', 'U') IS NOT NULL
-	DROP TABLE @resultsDatabaseSchema.@tableName;
+IF OBJECT_ID('@tableName', 'U') IS NOT NULL
+	DROP TABLE @tableName;
```

Comment:
Why would we drop the schema placeholder? I would like to be able to specify where the results tabel is created.

Reply:
This is to  fix failing unit tests.
In writeResultsTo.R, tableName is defined in lines 55-58 as concatenation of @resultDataBaseSchema, @writeTableName, and @cohortDefinitionID (if needed). So, followed similar approach in sqlOnly.R line 134 (but without cohort option, since it doesn't yet use that).

Reply:
I see, that should work then, thanks.